src/Networks/Blockcore.Networks.SeniorBlockCoin/SeniorBlockCoinSetup.cs

```diff
@@ -19,7 +19,7 @@ internal class SeniorBlockCoinSetup
             PremineReward = 256000000,
             PoWBlockReward = 100,
             PoSBlockReward = 100,
-            LastPowBlock = 35000,
+            LastPowBlock = 25000,
```

Comment:
You can just change that as its a consensus rule change, you need to handle with care.